synapse/lib/snap.py

```diff
@@ -303,7 +306,10 @@ def setUser(self, user):
         try:
 
             fnib = self._getNodeFnib(name, valu)
-            return await self._addNodeFnib(fnib, props=props)
+            print(f'>{fnib[0].name}={fnib[1]}', flush=True)
```

Comment:
should these be logger calls or removed?

Reply:
they should be nuked.

Reply:
:atomic: